File: app/src/main/java/me/imzack/app/end/view/fragment/ThanksFragment.java
package me.imzack.app.end.view.fragment;

import android.os.Bundle;
import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import me.imzack.app.end.R;
import me.imzack.app.end.view.adapter.LibraryListAdapter;

import butterknife.BindView;
import butterknife.ButterKnife;

public class ThanksFragment extends BaseFragment {

    @BindView(R.id.list_library)
    RecyclerView mLibraryList;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_thanks, container, false);
    }

    @Override
    public void onViewCreated(View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        ButterKnife.bind(this, view);

        mLibraryList.setAdapter(new LibraryListAdapter(getActivity()));
        mLibraryList.setHasFixedSize(true);
    }

    @Override
    public void onDetach() {
        super.onDetach();
    }
}

File: app/src/main/kotlin/me/imzack/app/end/view/fragment/ThanksFragment.kt
package me.imzack.app.end.view.fragment

import android.os.Bundle
import android.support.v7.widget.RecyclerView
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import butterknife.BindView
import butterknife.ButterKnife
import me.imzack.app.end.R
import me.imzack.app.end.view.adapter.LibraryListAdapter

class ThanksFragment : BaseFragment() {

    @BindView(R.id.list_library)
    lateinit var mLibraryList: RecyclerView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?) =
            inflater.inflate(R.layout.fragment_thanks, container, false)!!

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        ButterKnife.bind(this, view)

        mLibraryList.adapter = LibraryListAdapter(activity)
        mLibraryList.setHasFixedSize(true)
    }

    override fun onDetach() {
        super.onDetach()
    }
}
